modules/renter/hostdb/consts.go

```diff
@@ -0,0 +1,36 @@
+package hostdb
+
+import (
+	"time"
+
+	"github.com/NebulousLabs/Sia/build"
+)
+
+const (
+	defaultScanSleep = 1*time.Hour + 37*time.Minute
+	maxScanSleep     = 4 * time.Hour
+	minScanSleep     = 1*time.Hour + 20*time.Minute
+
+	maxSettingsLen = 4e3
+
+	hostRequestTimeout = 60 * time.Second
+	hostScanDeadline   = 60 * time.Second
+)
+
+var (
+	// hostCheckupQuantity specifies the number of hosts that get scanned every
+	// time there is a regular scanning operation.
+	hostCheckupQuantity = build.Select(build.Var{
+		Standard: int(250),
+		Dev:      int(4),
+		Testing:  int(5),
```

Comment:
I am reconsidering these values. I wanted them to be low enough that in testing we could surpass them with online hosts. I'm not actually sure how many hosts we can reasonably run simultaneously on a single machine at this time.